### ISSUE TITLE
multiple definition of `gfx_rend_ifp'

### ISSUE THREAD

**Issue by Kazade** [washingtondc-emu/washingtondc#85]
```
/usr/bin/ld: ../libwashdc/libwashdc.a(gfx.c.o):(.bss+0x0): multiple definition of `gfx_rend_ifp'; ../libwashdc/libwashdc.a(rend_common.c.o):(.bss+0x0): first defined here
/usr/bin/ld: ../libwashdc/libwashdc.a(gfx_tex_cache.c.o):(.bss+0x0): multiple definition of `gfx_rend_ifp'; ../libwashdc/libwashdc.a(rend_common.c.o):(.bss+0x0): first defined here
/usr/bin/ld: ../libwashdc/libwashdc.a(gdrom.c.o):(.data+0x0): multiple definition of `gdrom_reg_intf'; ../libwashdc/libwashdc.a(dreamcast.c.o):(.bss+0xe0): first defined here
/usr/bin/ld: ../libwashdc/libwashdc.a(gfx.c.o):(.bss+0x0): multiple definition of `gfx_rend_ifp'; ../libwashdc/libwashdc.a(rend_common.c.o):(.bss+0x0): first defined here
/usr/bin/ld: ../libwashdc/libwashdc.a(gfx_tex_cache.c.o):(.bss+0x0): multiple definition of `gfx_rend_ifp'; ../libwashdc/libwashdc.a(rend_common.c.o):(.bss+0x0): first defined here
/usr/bin/ld: ../libwashdc/libwashdc.a(gdrom.c.o):(.data+0x0): multiple definition of `gdrom_reg_intf'; ../libwashdc/libwashdc.a(dreamcast.c.o):(.bss+0xe0): first defined here
collect2: error: ld returned 1 exit status
```

Happens on Fedora 32 beta / GCC 10.0.1

**Comment by snickerbockers** [comment]
it should work now, let me know if there are any more problems.

**Comment by snickerbockers** [comment]
fixed in commit b4f220b0e682e36fc1055f7f5b89186edadfc55c
